machine/dni/dni_3048up/kernel/platform-dni_3048up.patch

```diff
@@ -0,0 +1,1933 @@
+From 8cdecead792e797e8ecaf8f50f18cbec08d56a9e Mon Sep 17 00:00:00 2001
+Subject: [PATCH] Support for Agema AG6248C-POE platform (ARM Cortex-A9 /
+ Helix4) networking platform Baseline files sourced from
+ Broadcom LDK 3.4.10
+
+diff --git a/arch/arm/boot/dts/agema_ag6248c-poe.dts b/arch/arm/boot/dts/agema_ag6248c-poe.dts
```

Comment:
Looks like the patch is copy from AG6248C-POE platform  patch sent by David Hu which seems to be NAND based 
Is DNI 3048UP same as AG6248C-POE or different platforms.
Can you please fix the platform name references to DNI 3048UP